storage-proofs/post/Cargo.toml

```diff
@@ -34,6 +34,7 @@ tempfile = "3"
 pretty_assertions = "0.6.1"
 rand_xorshift = "0.2.0"
 filecoin-hashers = { path = "../../filecoin-hashers", version = "1.0.0", default-features = false, features = ["poseidon", "sha256", "blake2s"]}
+pretty_env_logger = "0.4.0"
```

Comment:
I'd use `fil_logger` instead for consistency with other crates.

Reply:
I forgot to add. If `fil_logger` doesn't cut it for some reason, I'd rather fix fil_logger. Having the same logger across a larger system really is beneficial (I've been there in the past, with different log structures from different sub-subsystems running within the same daemon).

Reply:
Yeah it doesn't work for tests, because `fil_logger::init()` panics if it is called multiple times

Reply:
Looks like I need to check what pretty_env_logger is doing there :)